_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,110 @@
+  - A new *gRPC server*, more flexible and easier to maintain
+  - A *revamped dev UI*
+  - <Your extension awesome feature/improvements here>
+
+
+## Java Target 
+
+Quarkus 3 continues (like Quarkus 2) to target minimal version of Java 11. The default recommendation for users will be to use Java 17. 
+
+We will continue to revise minimal Java version based on user feedback and uptake of Java 17 in the ecosystem. For now we feel it is of higher value to allow as many Java 11 users as possible to upgrade to Quarkus 3 and use the Jakarta based dependencies. 
+
+## Release Plan
+
+We will continue the monthly cadence release of the 2.x version of Quarkus. Quarkus 3.x preview (alphas, betas) will be released in parallel. This blog marks the start of the first Quarkus 3 Alpha.
+
+It will take more than a month to integrate the change set we want and collect feedback (from users, and extension maintainers...) from the previous release. Thus, 3.x release cadence will be a bit slower than usual.
+
+Thus the intent is to have a time period of some months to allow continuous integration and start releasing a Quarkus Platform with the members that are ready to do and then continuously include more.
+
+Proposed plan:
+
+  - Start doing 3.0.0.Alpha core and platform releases at regular intervals to allow easy integration builds, starting first from 2.13.3 in the coming week
+  - 2.x releases will continue to happen on a monthly basis
+  - Announce Quarkus 3 on Quarkus blog when 3.0.0.Alphas start being useful for users to try (This Blog!)
+  - Feedback from and Outreach to extension contributors and users on readiness and challenges to help and adjust
+  - If all goes well, aim for a 3.0.0.Final in February 2023
+
+== Breakage? Keep calm!
+
+While the major version bump signals possible breaking changes, we still ask that all contributors still aim for a smooth upgrade experience. Changes like the move to Jakarta with its package renames have a big impact. We intend on doing what we can to make it so current extensions and user applications can migrate from Quarkus 2 to Quarkus 3 by providing tooling to perform the migration.
+
+Especially for the core API’s we will aim to allow such migration to be trivial and thus can be automated in a similar manner as how we’ve done automatic conversion of Quarkus 2.x to use Jakarta named packages.
+
+== Trying out Quarkus 3
+
+For now the easiest way to get started is using https://code.quarkus.io/?S=io.quarkus.platform%3A3.0[code.quarkus.io] or use the Quarkus CLI:
+
+```
+quarkus app create --stream=3.0
+```
+
+Mind you that at this stage, while all the core extensions are available, only parts of the Quarkus platform is present. Not all extensions have yet migrated to Jakarta packages (e.g. Camel Quarkus or Kogito are not available).
+
+-- TBD: mention the extensions already committed or planning to be ready.
```

Comment:
This probably needs to be removed?

Reply:
yeah - we have 3-4 to mention but better to mention in alpha2 where they had time to release against 3